.github/workflows/ci.yml

```diff
@@ -17,6 +17,20 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
 
+    services:
+      postgres:
+        image: "postgres"
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: fractal_test
```

Comment:
Is it reasonable to make this `fractal_client_test`? This way we'd avoid overlap with the backend CI